onyo/lib/onyo.py

```diff
@@ -812,25 +815,31 @@ def _mkdir_sanitize(self, dirs: Iterable[Union[Path, str]]) -> set[Path]:
 
         # errors
         if error_exist:
-            log.error('The following paths already exist:\n' +
-                      self._n_join(error_exist) + '\n' +
-                      'No directories were created.')
-            raise FileExistsError('The following paths already exist:\n' +
-                                  self._n_join(error_exist))
+            log.error(
+                'The following paths already exist:\n{}\nNo directories were '
+                'created.'.format('\n'.join(map(str, error_exist))))
+            raise FileExistsError(
+                'The following paths already exist:\n{}'.format(
+                    '\n'.join(map(str, error_exist))))
 
         if error_path_protected:
-            log.error('The following paths are protected by onyo:\n' +
-                      self._n_join(error_path_protected) + '\n' +
-                      'No directories were created.')
-            raise OnyoProtectedPathError('The following paths are protected by onyo:\n' +
-                                         self._n_join(error_path_protected))
+            log.error(
+                'The following paths are protected by onyo:\n{}\nNo '
+                'directories were created.'.format(
+                    '\n'.join(map(str, error_path_protected))))
+            raise OnyoProtectedPathError(
+                'The following paths are protected by onyo:\n{}'.format(
+                    '\n'.join(map(str, error_path_protected))))
 
         return dirs_to_create
 
     #
     # MV
     #
-    def mv(self, sources: Union[Iterable[Union[Path, str]], Path, str], destination: Union[Path, str], dryrun: bool = False) -> list[tuple[str, str]]:
+    def mv(
+            self, sources: Union[Iterable[Union[Path, str]], Path, str],
```

Comment:
Why is this on a new line? This can be on the same line as `def mv(`

Reply:
For a method as short as `mv` it doesn't make much of a difference, but for methods with longer names this pushes line-breaks further to the right. For example:

```
def this_has_a_really_long_name(self, foo, bar, baz,
                                lorem, ipsum, dolor,
                                sit, amet) -> bool:
    # code goes here
    return True
```
vs
```
def this_has_a_really_long_name(
        self, foo, bar, baz, lorem, ipsum, dolor, 
        sit, amet) -> bool:
    # code goes here
    return True
```

The reason to do this with `mv` is consistency.

Reply:
I know this PR is just merged, but want to add my two cents anyways:

I think there should not be the consistent rule "break lines for functions because they are sometimes long", but rather we should aim at choosing consistently the easiest readable solution.

I myself am still on the search for when to break lines and when not to do that (see some of my last few PRs), but in this case I think it is obviously not an improvement.